.golangci.yml

```diff
@@ -75,6 +75,16 @@ issues:
 linters-settings:
   depguard:
     rules:
+      no-sdk-instrumentation:
+        files:
+          - "**/*/{bridges,instrumentation}/**/*.go"
+          - "!**/*test/*.go"
+          - "!**/*test/**/*.go"
+          - "!**/*example/*.go"
+          - "!**/*example/**/*.go"
```

Comment:
Can we omit the instrumentation/runtime and bridges/prometheus here instead of adding lots of nolint comments?  Metric bridges are expected to rely on the SDK.